randovania/games/am2r/json_data/Hydro Station.txt

```diff
@@ -371,20 +381,26 @@ Extra - map_name: rm_a2a05
   * Layers: default
   * Morph Ball Tunnel to Hydro Station Exterior/Tunnel to Inner Alpha Nest North
   > Pickup (Missile Tank)
-      Trivial
+      Morph Ball
 
 > Dock to Varia Chamber Access Water Pipe; Heals? False
   * Layers: default
   * Open Passage to Varia Chamber Access Water Pipe/Dock to Inner Alpha Nest North
   > Dock to Hydro Station Exterior
-      Can Use Any Bombs
+      All of the following:
+          Can Use Any Bombs
+          Any of the following:
+              Mid-Air Morph (Beginner) or Tunnel Climb
+              All of the following:
+                  # Movement directly from pipe to tunnel
+                  Movement (Beginner) and Disabled Entrance Rando
```

Comment:
shouldnt this also check for the dead metroid?

Reply:
this is for the right pipe and should not require any checks for the interior alpha alpha

Reply:
This is now the second time i make that mistake whoops.